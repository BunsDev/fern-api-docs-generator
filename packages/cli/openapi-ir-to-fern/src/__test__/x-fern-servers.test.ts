import { testConvertOpenAPI } from "./testConvertOpenApi";

describe("x-fern-servers", () => {
    testConvertOpenAPI("x-fern-servers", "openapi.yml");
});
