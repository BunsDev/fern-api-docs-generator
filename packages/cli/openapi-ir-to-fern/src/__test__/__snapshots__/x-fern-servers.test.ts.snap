// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`x-fern-servers x-fern-servers docs 1`] = `
{
  "definitionFiles": {
    "transcripts.yml": {
      "types": {
        "SpeechModel": {
          "docs": "The speech model to use for the transcription.",
          "type": "string",
        },
      },
    },
  },
  "packageMarkerFile": {},
  "rootApiFile": {
    "default-environment": "Production",
    "display-name": "Test extension \`x-fern-enum\` for enums",
    "environments": {
      "Production": {
        "urls": {
          "api": "https://api.example.com",
          "auth": "https://auth.example.com",
        },
      },
      "Sandbox": {
        "urls": {
          "api": "https://api-sandbox.example.com",
          "auth": "https://auth-sandbox.example.com",
        },
      },
    },
    "error-discrimination": {
      "strategy": "status-code",
    },
    "name": "api",
  },
}
`;

exports[`x-fern-servers x-fern-servers simple 1`] = `
{
  "definitionFiles": {
    "transcripts.yml": {
      "types": {
        "SpeechModel": {
          "docs": "The speech model to use for the transcription.",
          "type": "string",
        },
      },
    },
  },
  "packageMarkerFile": {},
  "rootApiFile": {
    "default-environment": "Production",
    "display-name": "Test extension \`x-fern-enum\` for enums",
    "environments": {
      "Production": {
        "urls": {
          "api": "https://api.example.com",
          "auth": "https://auth.example.com",
        },
      },
      "Sandbox": {
        "urls": {
          "api": "https://api-sandbox.example.com",
          "auth": "https://auth-sandbox.example.com",
        },
      },
    },
    "error-discrimination": {
      "strategy": "status-code",
    },
    "name": "api",
  },
}
`;
